certification/internal/shell/skopeo.go

```diff
@@ -0,0 +1,27 @@
+package shell
+
+import (
+	"bytes"
+	"os/exec"
+	"strings"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/cli"
+)
+
+type SkopeoCLIEngine struct{}
+
+func (e SkopeoCLIEngine) ListTags(image string) (*cli.SkopeoReport, error) {
+	imageName := strings.Split(image, ":")[0]
+	cmd := exec.Command("skopeo", "list-tags", "docker://"+imageName)
+
+	var stdout, stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	err := cmd.Run()
+
+	if err != nil {
+		return nil, err
+	}
+
+	return &cli.SkopeoReport{Stdout: stdout.String(), Stderr: stderr.String()}, nil
```

Comment:
Since we know this is json data, I'd rather see the unmarshal take place here, and return structured data in SkopeoReport. Also, SkopeoReport should be SkopeoListTagsReport, as there might be others. The Report struct should still contain the raw data in stdout/stderr, but structuring the data before returning it will make for a cleaner interface.